cloud/shared/bin/lib/config_loader.py

```diff
@@ -63,27 +64,34 @@ def get_env_variables(self, variable_definitions: dict):
     # TODO: we do not validate type of the variable as we only have
     # strings currently. If we add non-strings, will need to validate
     def _validate_config(self, variable_definitions: dict, configs: dict):
-        is_valid = True
         validation_errors = []
 
         for name, definition in variable_definitions.items():
-            is_required = definition.get("required", False)
+            is_required = definition.get('required', False)
             config_value = configs.get(name, None)
 
             if is_required and config_value is None:
-                is_valid = False
-                validation_errors.append(
-                    f"{name} is required, but not provided")
+                validation_errors.append(f'[{name}] required, but not provided')
```

Comment:
Good suggestion! Rather than returning, I continue in order to process additional variable definitions. This also allowed me to remove the if config_value is not None checks below.